js/galleryfileaction.js

```diff
@@ -210,6 +216,13 @@ $(document).ready(function () {
 	var url = window.galleryFileAction.buildGalleryUrl('config', '', {extramediatypes: 1});
 	$.getJSON(url).then(function (config) {
 		window.galleryFileAction.buildFeaturesList(config.features);
+
+		if (config.mediatypes.length === 0) {
```

Comment:
Let me give you an example.
Afaik, the slideshow doesn't work on public shares on 9.0 because the scripts are not loaded any more, so the user sees thumbnails, but clicking on them does nothing.
Let's say scripts are still loaded, but there is a defect which prevents the proper list of supported media types to be sent.
With your patch, the user would have no idea that something is broken, because he would see the thumbnails and get some kind of preview, but only for a subset of the supported media types.
So it just makes debugging a bit harder and I think this could be avoided by properly letting the frontend know that  previews have been disabled.
Doing it in the config controller also has the advantage of sending the same message to the Gallery side, making it easy to print a different message or a notification to the user.